packages/ws-server/tests/acceptance/estimateGas.spec.ts

```diff
@@ -21,9 +21,9 @@
 // external resources
 import WebSocket from 'ws';
 import { expect } from 'chai';
-import { Contract, ethers, WebSocketProvider } from 'ethers';
-import basicContractJson from '@hashgraph/json-rpc-server/tests/contracts/Basic.json';
-import { AliasAccount } from '@hashgraph/json-rpc-server/tests/clients/servicesClient';
+import basicContractJson from '../../contracts/Basic.json';
```

Comment:
Since the folder is moved to ws-server, this path can't be found.  
```suggestion
import basicContractJson from '@hashgraph/json-rpc-server/tests/contracts/Basic.json';
```